File: dart/lib/model/v3_stop_point.dart
part of swagger.api;

class V3StopPoint {
  
  int stopId = null;

  V3StopPoint();

  @override
  String toString() {
    return 'V3StopPoint[stopId=$stopId, ]';
  }

  V3StopPoint.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    stopId = json['stop_id'];
  }

  Map<String, dynamic> toJson() {
    return {
      'stop_id': stopId
     };
  }

  static List<V3StopPoint> listFromJson(List<dynamic> json) {
    return json == null ? new List<V3StopPoint>() : json.map((value) => new V3StopPoint.fromJson(value)).toList();
  }

  static Map<String, V3StopPoint> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, V3StopPoint>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new V3StopPoint.fromJson(value));
    }
    return map;
  }
}

File: java-vertx/src/main/java/io/swagger/server/api/model/V3StopPoint.java
package io.swagger.server.api.model;

import java.util.Objects;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonCreator;
import io.vertx.core.json.JsonObject;
import io.vertx.codegen.annotations.DataObject;
/**
 * V3StopPoint
 */

@javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.JavaVertXServerCodegen", date = "2024-04-14T03:35:56.953023300+10:00[Australia/Sydney]")



public class V3StopPoint   {
  private Integer stopId = null;

  public V3StopPoint stopId(Integer stopId) {
    this.stopId = stopId;
    return this;
  }

  /**
   * Get stopId
   * @return stopId
   **/
    public Integer getStopId() {
    return stopId;
  }

  public void setStopId(Integer stopId) {
    this.stopId = stopId;
  }


  @Override
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    V3StopPoint v3StopPoint = (V3StopPoint) o;
    return Objects.equals(this.stopId, v3StopPoint.stopId);
  }

  @Override
  public int hashCode() {
    return Objects.hash(stopId);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class V3StopPoint {\n");
    
    sb.append("    stopId: ").append(toIndentedString(stopId)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(java.lang.Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}
